autoblocks/_impl/testing/models.py

```diff
@@ -13,6 +31,26 @@ class Threshold:
     gte: Optional[float] = None
 
 
+@dataclasses.dataclass()
+class EventEvaluation:
```

Comment:
if this is the schema returned by `evaluate_event`, this is what the end user will interact with. we shouldn't ask them to attach the `id` and `evaluator_external_id` themselves, we should that ourselves behind the scenes